ballerina/client.bal

```diff
@@ -52,7 +52,7 @@ public isolated client class Client {
 
     # Executes the query, which may return multiple results.
     #
-    # + sqlQuery - The SQL query such as `` `SELECT * from Album WHERE name={albumName}` ``
+    # + sqlQuery - The SQL query such as `` `SELECT * from Album WHERE name=${albumName}` ``
```

Comment:
Lets update the changelog

Reply:
Updated [e4b92b6](https://github.com/ballerina-platform/module-ballerinax-postgresql/pull/282/commits/e4b92b6ec02bb3345cbeb39a5c3bf80facc74e11)